TombRaider1996/TR1.csproj

```diff
@@ -36,7 +36,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="LiveSplit.Core">
-      <HintPath>..\..\..\vendor\LiveSplit\LiveSplit\bin\Release\LiveSplit.Core.dll</HintPath>
+      <HintPath>..\..\..\..\Downloads\LiveSplit\LiveSplit.Core.dll</HintPath>

```

Comment:
Could we add a second HintPath and resolve this for the future? Unsure if that would work, would require VS research and experimentation.

Reply:
Doesn't break anything on my end if I have both HintPaths in, so I'll edit accordingly.